### ISSUE TITLE
An item with the same key has already been added. Key: Hangar410

### ISSUE THREAD

**Issue by itsTKRaz** [tpill90/epic-lancache-prefill#62]
This error occurs when I try to `select-apps`. Tried to reset and clear account folder and re-authorize multiple times.
```[3:18:23 PM] Refresh token has expired!  EpicPrefill will need to login again...
[3:18:23 PM] Please login into Epic via your browser
[3:18:23 PM] If the web page did not open automatically, please manually open the following URL: https://legendary.gl/epiclogin
[3:18:29 PM] Retrieving owned apps
[3:20:40 PM] System.ArgumentException: An item with the same key has already been added. Key: Hangar410
   at System.Collections.Generic.Dictionary`2.TryInsert(TKey, TValue, InsertionBehavior)
   at EpicPrefill.Handlers.EpicGamesApi.<>c__DisplayClass9_0.<<LoadAppMetadataAsync>b__1>d.MoveNext()
--- End of stack trace from previous location ---
   at Spectre.Console.Progress.<>c__DisplayClass27_0.<<StartAsync>b__0>d.MoveNext() in Spectre.Console\Live\Progress\Progress.cs:line 98
--- End of stack trace from previous location ---
   at Spectre.Console.Progress.<>c__DisplayClass28_0`1.<<StartAsync>b__0>d.MoveNext() in Spectre.Console\Live\Progress\Progress.cs:line 133
--- End of stack trace from previous location ---
   at Spectre.Console.Internal.DefaultExclusivityMode.RunAsync(Func`1 func) in Spectre.Console\Internal\DefaultExclusivityMode.cs:line 40
   at Spectre.Console.Progress.StartAsync(Func`2 action) in Spectre.Console\Live\Progress\Progress.cs:line 116
   at Spectre.Console.Progress.StartAsync(Func`2 action) in Spectre.Console\Live\Progress\Progress.cs:line 96
   at EpicPrefill.Handlers.EpicGamesApi.LoadAppMetadataAsync(List`1 apps)
   at EpicPrefill.Handlers.EpicGamesApi.GetOwnedAppsAsync()
   at EpicPrefill.EpicGamesManager.GetAllAvailableAppsAsync()
   at EpicPrefill.CliCommands.SelectAppsCommand.BuildTuiAppModelsAsync(EpicGamesManager epicManager)
   at EpicPrefill.CliCommands.SelectAppsCommand.ExecuteAsync(IConsole console)
```

**Comment by tpill90** [comment]
Duplicate of #58

**Comment by tpill90** [comment]
This issue should now be resolved in the latest [v2.1.0](https://github.com/tpill90/epic-lancache-prefill/releases/tag/v2.1.0) release.  Please give it a test and let me know if the issue still persists on the ticket #58 since I believe that this issue is a duplicate of that one.  Thank you
